superset/connectors/druid/models.py

```diff
@@ -1363,7 +1364,7 @@ def homogenize_types(df: pd.DataFrame, groupby_cols: Iterable[str]) -> pd.DataFr
         Here we replace None with <NULL> and make the whole series a
         str instead of an object.
         """
-        df[groupby_cols] = df[groupby_cols].fillna("<NULL>").astype("unicode")
+        df[groupby_cols] = df[groupby_cols].fillna(NULL_STRING).astype("unicode")
```

Comment:
Somewhat unrelated to your change but we probably don't need the `.astype("unicode")` as in Python 3 all strings are unicode.

Reply:
That might be fine, but I don't know pandas well enough to be sure and I don't really think it's related to this PR.